src/test/java/org/apache/datasketches/memory/internal/BufferTest.java

```diff
@@ -234,14 +233,14 @@ public void checkPutGetArraysHeap() {
     for (int i = 0; i < n; i++) { arr[i] = i; }
 
     WritableBuffer wbuf = WritableMemory.allocate(n * 8).asWritableBuffer();
-    wbuf.putLongArray(arr, 0, n);
-    long[] arr2 = new long[n];
-    wbuf.resetPosition();
-    wbuf.getLongArray(arr2, 0, n);
-    for (int i = 0; i < n; i++) {
-      assertEquals(arr2[i], i);
+      wbuf.putLongArray(arr, 0, n);
```

Comment:
seems like some problem with alignment all over the place

Reply:
This occurred in a bunch of files.  I went through all of them and I think it is OK now.